packages/gatsby/src/schema/create-key.js

```diff
@@ -1,3 +1,3 @@
 const regex = new RegExp(`[^a-zA-Z0-9_]`, `g`)
 
-module.exports = key => key.replace(regex, `___`)
+module.exports = key => key.replace(`@`, ``).replace(regex, `___`)
```

Comment:
@jquense I've cherry-pick your two commits in my branch and…
![image](https://user-images.githubusercontent.com/1761469/27009910-bd188a9c-4e99-11e7-8b0c-4955aba086ef.png)

It's not working for me…